in-depth/submission_guidelines.md

```diff
@@ -86,6 +86,10 @@ Heading
 
 : The name of your theme or plugin.
 
+View
+
+: Choose between the image oriented `jumbovision` layout, or the text-oriented `card` layout.  The default for Themes is `jumbovision`, while plugins use `card`.  This option only needs to be specified if you'd like to use the opposite layout.
```

Comment:
Yeah, I more or less agree.  I just noticed I hadn't written anything about those two items, and decided to add something quickly.  I meant to follow it up with a comment asking your opinion, but I've been dealing with other things and forgot about it. :unamused: